basics/if.md

```diff
@@ -311,6 +311,7 @@ you are asked to "fix code", feel free to add missing blank lines.
     `Access denied` or `You didn't enter anything` depending on whether
     the user entered the correct password, a wrong password, or nothing
     at all by pressing Enter without typing anything.
+7. Make a programme to take the input from the user and check if it is a palindrome.
```

Comment:
Consider putting this exercise to the "Handy stuff with strings" section instead. Currently it doesn't explain how to reverse a string, but I can add that later.